src/EnergyPlus/RoomAirModelAirflowNetwork.hh

```diff
@@ -67,9 +67,10 @@ namespace RoomAirModelAirflowNetwork {
     public:
         int ZoneNum;
         int RoomAirNode;
+        int numAirDistUnits;
 
         // constructor
-        RAFNData() : ZoneNum(0), RoomAirNode(0)
+        RAFNData() : ZoneNum(0), RoomAirNode(0), numAirDistUnits(0)
```

Comment:
Add a new variable in the module level for unit test

Reply:
@lgu1234 I'm not super sure about this new variable. Does it have a use other than for unit testing?

Unrelated to this new variable, it looks like the `class` should be a `struct` since everything is public, probably can leave that for now.

Reply:
@jasondegraw The module level variable of numAirDistUnits is not necessary for unit test. I made it as a local variable and still pass the unit test. The code modification was uploaded a while ago. Please check.